data/Localization/german_(germany)/global.ini

```diff
@@ -8302,6 +8302,65 @@ Heads_Male_06=Gesicht 06
 Heads_Male_07=Gesicht 07
 Heads_Male_08=Gesicht 08
 Heads_Male_09=Gesicht 09
+HexPenetrator_Bounty_Obj_01_Long=Gehe zur letzten bekannten Position und neutralisiere das Ziel
+HexPenetrator_Bounty_Obj_02_Long=Hebe den Verschlüsselungsschlüssel auf
```

Comment:
Line 8306
Instead of *Verschlüsselungsschlüssel* i would suggest *Dechiffrierschlüssel*

Reply:
Encryption Key, not Decryption Key